### PR TITLE
Make `protoc-gen-combo` actually report errors

### PR DESCRIPTION
**Pull request by tamird**
Currently `protoc-gen-combo` always exits 0.

### CODE REVIEW

File: protoc-gen-gogo/protoc-gen-combo/combo.go [hunk 1]
```diff
@@ -70,8 +73,22 @@ func (this MixMatch) Gen(folder string, news []string) {
 	var regenerate = exec.Command("protoc", args...)
 	out, err := regenerate.CombinedOutput()
 	if err != nil {
-		fmt.Printf("%s\n", string(out))
-		panic(err)
+		fmt.Print("protoc-gen-combo: error:", err)
```
Comment (awalterschulze) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: The out variable provides more information than the err variable, when an error occurs.

File: protoc-gen-gogo/protoc-gen-combo/combo.go [hunk 2]
```diff
@@ -70,8 +73,22 @@ func (this MixMatch) Gen(folder string, news []string) {
...
+		os.Exit(1)
+	}
+
+	failed := false
+	scanner := bufio.NewScanner(bytes.NewReader(out))
+	for scanner.Scan() {
```
Comment (awalterschulze) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: I don't want to fail on WARNING, not yet anyway.

**Comment by awalterschulze** [comment]
Maybe I should ask why did you come to a point where you wanted to make this change?

File: protoc-gen-gogo/protoc-gen-combo/combo.go (hunk 2 above)
Comment (tamird) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: can you read it again? it doesn't fail on warning.

File: protoc-gen-gogo/protoc-gen-combo/combo.go (hunk 1 above)
Comment (tamird) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: i think they are actually mutually exclusive.

**Comment by tamird** [comment]
I was trying to fix #98 and discovered that this binary silently exits 0 when protoc fails.

File: protoc-gen-gogo/protoc-gen-combo/combo.go (hunk 2 above)
Comment (awalterschulze) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: What is wrong with me. Sorry about that.

File: protoc-gen-gogo/protoc-gen-combo/combo.go (hunk 1 above)
Comment (awalterschulze) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: Could you maybe move the scanner that prints out the text to above the error check.
That way error or no error the output gets printed out and if there is any type of error os.Exit(1) is called.

Comment (tamird) [inline protoc-gen-gogo/protoc-gen-combo/combo.go]: done

**Comment by awalterschulze** [comment]
Perfect :)

**Pull request merged**